<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 251, 198);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>371</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Welcome to Helpdesk Ticketing system </string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 206, 178);</string>
    </property>
    <property name="title">
     <string>Login with your Credential </string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_username">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_login">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AdminLoginButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Admin Login </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_2"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit </string>
    </property>
    <addaction name="actionSave_3"/>
    <addaction name="actionPaste"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Us"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionDashboard"/>
    <addaction name="actionCustomer"/>
    <addaction name="actionEmail"/>
    <addaction name="actionKnowledge"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
   <addaction name="menuSystem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 0);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionHome"/>
   <addaction name="separator"/>
   <addaction name="actionDashboard"/>
   <addaction name="separator"/>
   <addaction name="actionCustomer"/>
   <addaction name="separator"/>
   <addaction name="actionEmail"/>
   <addaction name="separator"/>
   <addaction name="actionKnowledge"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Open.png</normaloff>:/Images/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Save.png</normaloff>:/Images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Exit.png</normaloff>:/Images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave_3">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Copy.png</normaloff>:/Images/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Paste.png</normaloff>:/Images/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Undo.png</normaloff>:/Images/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Redo.png</normaloff>:/Images/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/About.png</normaloff>:/Images/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/About.png</normaloff>:/Images/About.png</iconset>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Home.png</normaloff>:/Images/Home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionDashboard">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/DashBoard.png</normaloff>:/Images/DashBoard.png</iconset>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </action>
  <action name="actionCustomer">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Customer.png</normaloff>:/Images/Customer.png</iconset>
   </property>
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Email.png</normaloff>:/Images/Email.png</iconset>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </action>
  <action name="actionKnowledge">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Knowledge.png</normaloff>:/Images/Knowledge.png</iconset>
   </property>
   <property name="text">
    <string>Knowledge</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
